core/src/main/scala/org/apache/spark/scheduler/Stage.scala

```diff
@@ -92,6 +92,29 @@ private[spark] abstract class Stage(
    */
   private var _latestInfo: StageInfo = StageInfo.fromStage(this, nextAttemptId)
 
+  /**
+   * Set of IDs of stage attempts that have failed with a FetchFailure. We keep track of these
```

Comment:
`IDs of stage attempts` is a little confusing. Can you just call it `Set of stage attempt IDs`